notebooks/triposr-3d-reconstruction/triposr-3d-reconstruction.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.        'output_attentions': torch.tensor(False),
</pre><p>As we already discussed early, it is better to avoid using bool constants that should not be changed during pipeline runningas inputs, it is better to appear on default values for them (the reason is complication of model structure and blocking optimization due to unknown value of this input parameter in one cease and confusion for users who want to change this behaviour by these flags and not able to do that due to pytorch tracing workflow with resolving control-flow using input data path in other cases. Please export model without having them in example_input</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Removed</p>